CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 * [#4741](https://github.com/bbatsov/rubocop/issues/4741): Make `Style/SafeNavigation` correctly exclude methods called without dot. ([@drenmi][])
 * [#4740](https://github.com/bbatsov/rubocop/issues/4740): Make `Lint/RescueWithoutErrorClass` aware of modifier form `rescue`. ([@drenmi][])
 
+### Changes
+
+* [#4746](https://github.com/bbatsov/rubocop/pull/4746): The `Lint/InvalidCharacterLiteral` cop has been removed since it was preventing a warning that we were not sure if it can be emited by parsable ruby. ([@deivid-rodriguez][])
```

Comment:
I guess the message here should be simply "because it was never being actually triggered" (or something like this).